mockintosh/hbs/methods.py

```diff
@@ -46,7 +46,7 @@ def reg_ex(this, regex, *args, **kwargs):
 
 def json_path(this, text, path):
     data = text
-    if data is None:
+    if isinstance(data, tuple) and data[0] is None:
```

Comment:
Why do we need this check? `null` is a valid JSON document and `$` as JSONPath would succesfully match its root element.

Reply:
This was related to the change that I've made in here https://github.com/up9inc/pybars4/commit/3f294fc32e9c9034a73022cf37c7cee070338b04#diff-e3fbc2a2ce5876feb7c919471cdb1cefea61644a6e7ba15d61c2571ab71b5aa8R325